### PR TITLE
Stat-doc-fix

### PR DESCRIPTION
**Pull request by bh4t**
change example bucket names to simple names

### CODE REVIEW

File: cmd/mv-main.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ var mvCmd = cli.Command{
 USAGE:
   {{.HelpName}} [FLAGS] SOURCE [SOURCE...] TARGET
 
-FLAGS:
```
Comment (ravindk89) [inline cmd/mv-main.go:88]: I think this was unintentional :)

File: cmd/mv-main.go [hunk 2]
```diff
@@ -97,7 +97,7 @@ EXAMPLES:
       {{.Prompt}} {{.HelpName}} Music/*.ogg s3/jukebox/
 
   02. Move a folder recursively from MinIO cloud storage to Amazon S3 cloud storage.
-      {{.Prompt}} {{.HelpName}} --recursive play/mybucket/burningman2011/ s3/mybucket/
+      {{.Prompt}} {{.HelpName}} --recursive play/mybucket// s3/mybucket/
```
Comment (ravindk89) [inline cmd/mv-main.go:100]: double slash, guessing ctrl+f issue?

**Review by ravindk89 — CHANGES_REQUESTED** [review]
I am guessing the play-health.json needs to be dropped as well.

File: cmd/mv-main.go [hunk 3]
```diff
@@ -85,7 +85,7 @@ var mvCmd = cli.Command{
 USAGE:
   {{.HelpName}} [FLAGS] SOURCE [SOURCE...] TARGET
 
-FLAGS:
+myfolderFLAGS:
```
Comment (harshavardhana) [inline cmd/mv-main.go:88]:
```suggestion
FLAGS:
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by ravindk89 — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**